src/App.js

```diff
@@ -1,16 +1,83 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import ChatLog from './components/ChatLog';
 import chatMessages from './data/messages.json';
+import SelectColor from './components/SelectColor';
+
+// iterate through chatMessages and create a list of senders
+const senders = [];
+for (const message of chatMessages) {
+  if (!senders.includes(message.sender)) {
+    senders.push(message.sender);
+  }
+}
```

Comment:
Could pull this into a method and give it a descriptive name to convey what your code is doing instead of comments. Maybe something like getSenderList()